packages/app-desktop/gui/MasterPasswordDialog/Dialog.tsx

```
@@ -138,7 +141,10 @@ export default function(props: Props) {
 		if (event.cancelled) return;
 		setCurrentPasswordIsValid(isValid);
 	}, [currentPassword]);
-
+	
+	/**
```

ESLint

[error] Expected multiple line comments instead of a block comment.

(multiline-comment-style)